ff/serving/coordinator/coordinator_test.go

```diff
@@ -67,14 +77,15 @@ func TestCoordinatorCalls(t *testing.T) {
 	if err := testCoordinatorTrainingSet(); err != nil {
 		t.Fatalf("coordinator could not create training set: %v", err)
 	}
-	//g.Go(testCoordinatorCreateTransformation)
+	if err := testRegisterPrimaryTableFromSource(); err != nil {
```

Comment:
At some point, give change these to be subtests with individual names instead of having them output as a single test